test/functional/uuid/postgres/pgcrypto.ts

```diff
@@ -6,13 +6,16 @@ import {closeTestingConnections, createTestingConnections, reloadTestingDatabase
 import {Post} from "./entity/Post";
 import {Question} from "./entity/Question";
 
-describe("uuid-postgres", () => {
+describe("pgcrypto", () => {
```

Comment:
The diff isn't exactly clear what I did here - I actually renamed the `uuid-postgres` tests to `uuid-ossp`, and configured the tests to use the `uuid-ossp` extension. Then I copied the `uuid-ossp.ts` file to `pgcrypto.ts`, left all the tests the same, and set the extension to `pgcrypto`.